integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/VariantAnimationTest.kt

```diff
@@ -68,7 +74,38 @@ fun VariantAnimationTest() {
         }
     }
 
-    CompositionLocalProvider(LocalDesignDocSettings provides DesignDocSettings(useSquoosh = true)) {
+    CompositionLocalProvider(
+        LocalDesignDocSettings provides
+            DesignDocSettings(
+                useSquoosh = true,
+                customVariantTransition = { context ->
+                    if (context.fromComponentSet("RightPanelAndroid")) {
```

Comment:
Eventually we'll have some tooling to have the transitions come from the DCF file (and presumably from Figma eventually too, to close the loop), so this might be a short term API... I think this will be the first time we've exposed node ids directly to user code, and I'm wondering if we should have a warning annotation on these methods, or wrap the strings in a "UnstableNodeId" type or something like that so that it's clear what's going on, and that any design change can break the animation.

Reply:
Done